src/one_dragon_qt/view/installer_interface.py

```
@@ -32,21 +52,167 @@ def __init__(self, installer_dir: str | None, work_dir: str, parent=None):
+        copied_files: list[tuple[Path, Path, dict]] = []
+        for item in files:
+            if not isinstance(item, dict):
+                continue
+            rel = item.get('path')
+            if not rel or not isinstance(rel, str):
+                continue
+            rel_norm = rel.replace('\\', '/')
+            src_path = (src_root / rel_norm)
+            dst_path = (dst_root / rel_norm)
+
+            # 安全：只允许搬运 src_root 下的内容
```

Ruff (0.14.8)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)